hermes-client/src/main/java/pl/allegro/tech/hermes/client/metrics/MetricsHermesSender.java

```diff
@@ -27,6 +27,9 @@ public MetricsHermesSender(HermesSender sender, MetricRegistry metrics) {
             if (resp != null) {
                 metrics.counter(prefix + ".status." + resp.getHttpStatus()).inc();
             }
+            if (cause != null) {
+                metrics.counter(prefix + ".exception." + cause.getClass().getSimpleName()).inc();
```

Comment:
i dont think we should include exception name: i would rather inspect the cause and add metrics for some common exceptions like:

* `failure.timeout` (for timeout exceptions)
* `failure.other` (for unknown stuff)